### PR TITLE
Argument renaming

### PR DESCRIPTION
**Pull request by stijnvanhoey**
This mainly handles #112 by renaming the arguments. Output of R CMD CHECK:

```
checking installed package size ... NOTE
  installed size is 34.7Mb
  sub-directories of 1Mb or more:
    data      3.1Mb
    extdata  31.2Mb
R CMD check results
0 errors | 0 warnings | 1 note 
```
There are still some elements for discussion reported on #112, but I would already start reviewing and checking the new code @peterdesmet  and @adokter

### CODE REVIEW

File: R/map.R [hunk 1]
```diff
@@ -65,8 +68,27 @@ map <- function(x, ...) {
 #' @describeIn map plot a 'ppi' object on a map
 #' @export
 map.ppi <- function(x, map, param, alpha = 0.7, xlim, ylim,
-                    zlim = c(-20, 20), ratio, radar.size = 3,
-                    radar.color = "red", n.color = 1000, ...) {
+                    zlim = c(-20, 20), ratio, radar_size = 3, radar.size = 3,
```
Comment (peterdesmet) [inline R/map.R]: @stijnvanhoey arguments radar.size and n.color are still present

File: R/plot.vp.R [hunk 2]
```diff
@@ -24,11 +26,25 @@
 #' @examples
 #' data(example_vp)
 #' plot(example_vp)
-#' plot(example_vp, line.col = "blue")
-plot.vp <- function(x, quantity="dens",
+#' plot(example_vp, line_col = "blue")
+plot.vp <- function(x, quantity = "dens",
                     xlab = expression("volume density [#/km"^3*"]"),
-                    ylab = "height [km]", line.col='red', line.lwd=1, ...) {
+                    ylab = "height [km]", line_col = 'red', line.col = 'red',
```
Comment (peterdesmet) [inline R/plot.vp.R]: line.col and line.lwd still appear as arguments

File: R/plot.vpi.R [hunk 3]
```diff
@@ -46,15 +47,22 @@
 #' # plot the migration traffic rates
 #' plot(vpi)
 #' # plot the vertically integrated densities, without night shading:
-#' plot(vpi, quantity = "vid", nightshade = FALSE)
+#' plot(vpi, quantity = "vid", night_shade = FALSE)
 plot.vpi <- function(x, quantity = "mtr", xlab = "time",
                      ylab = "migration traffic rate [#/km/h]",
-                     main = "MTR", nightshade = TRUE, elev = -0.268,
-                     lat = NULL, lon = NULL, ylim = NULL, ...) {
+                     main = "MTR", night_shade = TRUE, nightshade = TRUE,
```
Comment (peterdesmet) [inline R/plot.vpi.R]: Both night_shade and nightshade

File: R/plot.vpts.R [hunk 4]
```diff
@@ -60,10 +64,37 @@
 #' # plot total reflectivity factor (rain, birds, insects together):
 #' plot(ts[1:500], ylim = c(0, 3000), quantity = "DBZH")
 plot.vpts <- function(x, xlab = "time", ylab = "height [m]", quantity = "dens",
-                      log = TRUE, barbs = TRUE, barbs.h = 10, barbs.t = 20,
-                      barbs.dens = 5, zlim, legend.ticks, main, ...) {
+                      log = TRUE, barbs = TRUE, barbs_height = 10, barbs.h = 10,
```
Comment (peterdesmet) [inline R/plot.vpts.R]: Still double parameters

File: R/pvolfile.R [hunk 5]
```diff
@@ -3,18 +3,27 @@
...
 #' otherwise FALSE
 #'
 #' @export
 #'
 #' @examples
 #' volume <- system.file("extdata", "volume.h5", package = "bioRad")
 #' is.pvolfile(volume)   #> TRUE
-is.pvolfile <- function(filename) {
-  type <- get_odim_object_type(filename)
+is.pvolfile <- function(file, filename = NULL) {
```
Comment (peterdesmet) [inline R/pvolfile.R]: Why is there still filename = NULL?

File: R/select_vpfiles.R [hunk 6]
```diff
@@ -19,14 +19,15 @@
 #' @examples
 #' my_path <- "~/my/directory/"
 #' select_vpfiles(my_path, "2016-10-01", "2017-01-31", c("be"))
-select_vpfiles <- function(path, start_date, end_date,
-                           country = NULL, radar = NULL) {
+select_vpfiles <- function(path, date_min, date_max, country = NULL,
```
Comment (peterdesmet) [inline R/select_vpfiles.R]: Shouldn't this be directory instead of path?

File: R/map.R (hunk 1 above)
Comment (adokter) [reply R/map.R]: I think this has to do with maintaining backward compatibility.  It is rather confusing though and I wonder whether we shouldn't just break backward compatibility (it's not that we have a very large user base) @peterdesmet what do you think

File: R/select_vpfiles.R (hunk 6 above)
Comment (adokter) [reply R/select_vpfiles.R]: yes you're right, directory is better as we use that elsewhere as well

File: R/map.R (hunk 1 above)
Comment (peterdesmet) [reply R/map.R]: Yeah, @stijnvanhoey told me later that it was for backward compatibility. I would drop it: including these and remembering to eventually remove them will be more confusing for us maintainers, than it being helpful to users. A wrong parameter error in old code is pretty straightforward to debug.

Comment (stijnvanhoey) [reply R/map.R]: see #126

File: R/plot.vp.R (hunk 2 above)
Comment (stijnvanhoey) [reply R/plot.vp.R]: see #126

File: R/plot.vpi.R (hunk 3 above)
Comment (stijnvanhoey) [reply R/plot.vpi.R]: see #126

File: R/plot.vpts.R (hunk 4 above)
Comment (stijnvanhoey) [reply R/plot.vpts.R]: see #126

File: R/pvolfile.R (hunk 5 above)
Comment (stijnvanhoey) [reply R/pvolfile.R]: see #126

**Pull request merged**